guides/common/modules/proc_cloning_satellite_server.adoc

```diff
@@ -6,6 +6,14 @@ Use the following procedures to clone your {ProjectServer}.
 
 The {Project} clone tool does not support migrating a {SmartProxyServer} to {EL} 8.
 Instead, you must backup the existing {SmartProxyServer}, restore it on {EL} 8, and then reconfigure {SmartProxyServer}.
+You can clone the {ProjectServer} to create instances for trying out upgrades and the migration of instances to a different machine or operating system.
+This is an optional step to provide more flexibility during the upgrade.
++
+[NOTE]
```

Comment:
I think you can move this note to a small snippet. I made a suggestions to the wording above.